src/libs/elektra/keymeta.c

```diff
@@ -194,21 +203,26 @@ const Key * keyNextMeta (Key * key)
 	return ret;
 }
 
-/**Returns the value of a meta-information which is current.
+/** Returns the Key of the metadata the internal iterator currently
+ * points at.
```

Comment:
```suggestion
/** Returns the metadata Key the internal iterator currently
 * points at.
```

Reply:
Seems a bit wonky to me, maybe
> Returns the metadata Key at the internal iterator's current position

is even better?

Reply:
Yes, sounds even better!